src/influx/index.js

```diff
@@ -49,78 +49,65 @@ const saveData = async (influxdb, url, measurement) => {
     }
 }
 
-const markSuccess = async (influxdb, url) => {
+const markPoints = async (influxdb, points, option) => {
     try {
-        const measurement = [ { measurement: 'success',
-            tags: { url: url },
-            fields: { success: true } } ];
-        const result = await influxdb.writePoints(measurement);
-        console.log(`Successfully marked ${influxdb.config.database} data for ${url} as success`);
+        const result = await influxdb.writePoints(points);
+        console.log(`Successfully saved data for ${option} into ${points[0].measurement}.`);
         return result;
     } catch (err) {
-        console.log(`Failed to save data for ${url}`, err);
-        return Promise.reject(`Failed to mark data into influxdb for ${url}`);
+        console.log(`Failed to mark point into ${points[0].measurement}`, err);
+        return Promise.reject(`Failed to mark data into influxdb for ${points[0].measurement}`);
     }
 }
 
+const markSuccess = async (url) => {
```

Comment:
I think it's confusing to leave these as async functions. Are these used anywhere in the actual plugins?